amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -117,7 +117,7 @@ For example, the test connection can fail if your credentials have a restricted
 [[connection]]
 === Connection (S3) Connection Type
 
-. Enter the following information on the *General* tab of the *Global Element Properties* screen to configure the Connection (S3) connection type:
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure the Connection (S3) connection type:
 +
```

Comment:
Double checking that the plus signs are intended to be showing. If not, they can just be deleted.